portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-validator.js

```diff
@@ -104,21 +107,24 @@ const validateCsvData = (csvData) => {
  */
 const validateFilenameContainsReportPeriod = (filename, dueReportPeriod) => {
```

Comment:
As we're now including other filename validations in this function we could make the name more generic, like `validateUtilisationReportFilename`?

Reply:
Changed to `validateFilenameFormat`